### PR TITLE
Release v0.0.1

### PR DESCRIPTION
**Pull request by CJWaszczuk**
### Added

- TypeScript support
- SASS support
- `lbh-frontend` library to style components
- `ErrorMessage` component
- `ComponentRegister` class for linking external components to library
  components
- `Link` component
- `Hint` component
- `Label` component
- `Tag` component
- `FieldsetLegend` component
- `Fieldset` component
- `InputButton` component
- `Button` component
- `PageAnnouncement` component

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -24,3 +26,5 @@ and this project adheres to
 - `Fieldset` component
 - `InputButton` component
 - `Button` component
+
+[0.0.1]: https://github.com/LBHackney-IT/lbh-frontend-react/releases/tag/v0.0.1
```
Comment (erbridge) [inline CHANGELOG.md]: Add an `[Unreleased]` link (and shange `# Unreleased` into `# [Unreleased]`.

File: package.json [hunk 2]
```diff
@@ -1,5 +1,6 @@
 {
   "name": "lbh-frontend-react",
+  "version": "0.0.1",
```
Comment (erbridge) [inline package.json]: Also update the `package-lock.json` (you can run `npm install` to automate this).

**Comment by erbridge** [comment]
Oh and rebase on top of the current `master`, please.

**Review by erbridge — APPROVED** [review]
I'll do the next bit.

File: README.md [hunk 3]
```diff
@@ -142,6 +142,57 @@ because Dependabot can get very noisy) have code owners:
 npm run lint:codeowners
 ```
 
+### Releasing versions
+
+1. Create a new branch called `release/vx.y.z`, where `x.y.z` is the new version
+   number, following [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+1. Update `CHANGELOG.md` to batch the changes in this version under a heading in
```
Comment (LBHELewis) [inline README.md]: Do you know you've got a lot of `1.`s throughout this section?

File: bin/bump-version [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+  exit 1
+fi
+
+ROOT_DIR=$(cd "$(dirname "$0")/.."; pwd -P)
+
+cd "$ROOT_DIR"
+
+CURRENT_BRANCH=$(git rev-parse --abbrev-ref HEAD)
+
+if ! [[ $CURRENT_BRANCH =~ ^release/v[0-9]+\.[0-9]+\.[0-9]+$ ]]; then
```
Comment (LBHELewis) [inline bin/bump-version]: Should we not check for `$CURRENT_BRANCH =~ ^release/v$NEW_VERSION`?

File: README.md (hunk 3 above)
Comment (erbridge) [reply README.md]: Yup. That's intentional. Markdown will do the ordering for us, so we don't need to mess with renumbering when adding or removing steps.

File: bin/bump-version (hunk 4 above)
Comment (erbridge) [reply bin/bump-version]: Yes. You're right. I added the regular expression matching after doing the first pass. I will fix this.

**Pull request merged**